eng/pipelines/templates/stages/1es-redirect.yml

```diff
@@ -31,8 +31,8 @@ extends:
           disableAutoBaselineOnNonDefaultBranches: true
           enableForGitHub: true
       sourceAnalysisPool:
```

Comment:
@raych1: Do you know how to test this?

Reply:
This is a pipeline using this template. Can you push the change to a branch in Azure spec repo so that it can be tested by specifying the branch while queuing a run?
https://dev.azure.com/azure-sdk/internal/_build?definitionId=7412

It should be fine if you have validated the image and pool work.

Reply:
I don't see any way to approve to use the new pool. Is it due to no permission for me?
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4537711&view=logs&s=ff05ad62-bb9a-53b6-ce9f-72f329a63e7c

Reply:
Looks like you got this working eventually.  I'll try to migrate the permissions when I create the new pool

Reply:
I queued a new run of the above pipeline and the result looks good.